packages/clerk-js/src/ui/elements/InformationBox.tsx

```diff
@@ -0,0 +1,26 @@
+import type { LocalizationKey } from '../customizables';
+import { Flex, Icon, Text } from '../customizables';
+import { InformationCircle } from '../icons';
+
+type InformationBoxProps = {
+  message: LocalizationKey | string;
+};
+
+export function InformationBox(props: InformationBoxProps) {
+  return (
+    <Flex
+      sx={t => ({
+        gap: t.space.$2,
+        padding: `${t.space.$3} ${t.space.$4}`,
+        backgroundColor: t.colors.$blackAlpha50,
+        borderRadius: t.radii.$md,
+      })}
+    >
+      <Icon icon={InformationCircle} />
+      <Text
+        localizationKey={props.message}
+        sx={t => ({ color: t.colors.$blackAlpha700 })}
```

Comment:
Can't we just use `colorScheme='inherit'` here?

Reply:
@desiprisg With `colorScheme='inherit'` this becomes black instead of gray.

<img width="605" alt="image" src="https://github.com/clerkinc/javascript/assets/5611795/702fa934-d5b0-49bd-a04b-eef4696cfb80">